src/app/services/bucket/baseBucketService.py

```diff
@@ -1,22 +0,0 @@
-from abc import ABC
-from abc import abstractmethod
-
```

Comment:
I would have kept it and dummyBucketService.
But let's not argue and let's move on.
OK for the deletion.